.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+
+<!-- 
+This template will help you to have a meaningful PR, please follow it and do not leave it blanket. 
+-->
+
+# PR Description 
+
+## Related Issue 
+
+<!-- 
+Please use this format to link other issues with thier numbers: Close #123 
```

Comment:
We could make a section for this like they have [here](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#which-issues-this-pr-fixes)

Reply:
Do you mean as copy change or something else?

Reply:
Here is the Markdown from Kubernetes: 
```md
#### Which issue(s) this PR fixes:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #
```

Reply:
I just mean having it outside of the comment too so it is less likely to be missed and that could also help it be more consistent, if that makes sense :sweat_smile: . We can definitely use our style of Close # (or Closes #)

Reply:
I got that! thank you. I will add it outside the comment too.